File: all/abc172/abc172_b/abc172_b.cs
using System;
public class A
{
  static void Main()
  {
    string s = Console.ReadLine();
    string t = Console.ReadLine();

    int ans = 0;
    for(int i = 0; i < s.Length; i++)
    {
      if(s[i] != t[i])
      {
        ans++;
      }
    }

    Console.WriteLine(ans);
  }
}

File: all/abc172/abc172_b/abc172_b.cpp
#include <bits/stdc++.h>
using namespace std;
int main() {
  int ans = 0;
  string s, t; cin >> s >> t;
  for(int i = 0; i < s.size(); i++) if(s[i] != t[i]) ans++;
  cout << ans;
}